ebnf/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+type scanner struct {
+	input []byte
+	ch    byte
+	pos   int
+
+	insideBraces int
+}
+
+func (s *scanner) scan() {
+	s.skipUntilStart()
```

Comment:
We skip all the header part of the grammar file because we only need the grammar rules.